src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -21,17 +21,14 @@
 
 <template>
 	<div class="wrapper">
-		<!-- Tooltip -->
-		<Popover trigger="hover" placement="bottom">
-			<Button slot="trigger"
-				class="toggle"
-				icon=""
-				:aria-label="t('spreed','Create a new group conversation')"
-				@click="showModal">
-				<Plus size="24" />
-			</Button>
-			<p>{{ t('spreed','Create a new group conversation') }}</p>
-		</Popover>
+		<Button slot="trigger"
```

Comment:
it's not a component, so lowercase please
```suggestion
		<button slot="trigger"
```